crates/server/src/auth/tls.rs

```diff
@@ -19,6 +19,8 @@ use rustls_pemfile::Item::{ECKey, PKCS8Key, RSAKey, X509Certificate};
 pub struct TrustedCertificate;
 
 #[repr(transparent)]
+#[allow(missing_debug_implementations)]
```

Comment:
Why?

Reply:
Well, the whole point of turning on that lint is that we don't know what consumers of the API might want to debug. If there are secrets, then obviously those shouldn't be printed. But if you disagree with the idea of adding Debug on every public item, then you shouldn't enable that lint in the first place. :P